content/docs/services/intro.md

```diff
@@ -5,12 +5,36 @@ menu:
 aliases: 
     - /docs/services/
 title: Introduction
-layout: services
 weight: -1
 ---
 
-cloud.gov offers multiple [managed services]({{< relref "docs/apps/managed-services.md" >}}) to allow your application to expand its functionality. You can also create [user-provided services]({{< relref "docs/apps/production-ready.md#protect-access-to-sensitive-credentials" >}}).
+You can expand the functionality of your cloud.gov application by making use of services. Before your application can use a service, the service must be provisioned, and credentials for using the service must be supplied to your application. 
 
-To list all the managed services and plans available to your organization, you can run `cf marketplace` from your command line.
+There are two ways to provision services:
```

Comment:
This list should mirror the set of sections below. Suggest making a bold keyword at the beginning of each list item that a reader can match to the section below (or making them links), to help the reader map the info.